src/Microsoft.Extensions.Configuration.AzureAppConfiguration/JsonConfigurationParser.cs

```diff
@@ -0,0 +1,108 @@
+﻿using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration
+{
+    internal static class JsonConfigurationParser
+    {
+        public static bool IsJsonContentType(string contentType)
+        {
+            try
+            {
+                string acceptedMainType = "application";
+                string acceptedSubType = "json";
+
+                ContentType ct = new ContentType(contentType.Trim().ToLower());
+                var type = ct.MediaType;
+                if (Equals(type, FeatureManagementConstants.ContentType)
+                    || Equals(type, KeyVaultConstants.ContentType))
+                {
+                    return false;
+                }
+
+                if (type.Contains('/'))
+                {
+                    string mainType = type.Split('/')[0];
+                    string subType = type.Split('/')[1];
+                    if (Equals(mainType, acceptedMainType))
```

Comment:
>Equals [](start = 24, length = 6)

This would ignore JSON parsing if the content type is `APPLICATION/JSON` (upper case).

Reply:
Just noticed you convert it to lowercase before using it.

---
In reply to: [453139448](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r453139448) [](ancestors = 453139448)